src/testing/utils.tsx

```diff
@@ -15,7 +15,26 @@ import configureStore from "redux-mock-store";
 import FormikForm from "app/base/components/FormikForm";
 import type { AnyObject } from "app/base/types";
 import type { RootState } from "app/store/root/types";
-import { rootState as rootStateFactory } from "testing/factories";
+import {
+  domainState as domainStateFactory,
+  fabric as fabricFactory,
+  fabricState as fabricStateFactory,
+  generalState as generalStateFactory,
+  podDetails as podDetailsFactory,
+  podState as podStateFactory,
+  podStatus as podStatusFactory,
+  powerType as powerTypeFactory,
+  powerTypesState as powerTypesStateFactory,
+  resourcePoolState as resourcePoolStateFactory,
+  rootState as rootStateFactory,
+  spaceState as spaceStateFactory,
+  subnet as subnetFactory,
+  subnetState as subnetStateFactory,
+  vlan as vlanFactory,
+  vlanState as vlanStateFactory,
+  zoneGenericActions as zoneGenericActionsFactory,
+  zoneState as zoneStateFactory,
+} from "testing/factories";
```

Comment:
I'm guessing we only need this base stuff here, makes sense since for tests that concern controllers/machines etc we'll need more granular control

Reply:
We can add more test cases in the future for particular network setup etc. but it should be a complete state with everything loaded. There will be a bunch of things missing here, but we'll keep adding as we go.

In the future this could work like this (just an idea): `getTestState("simple-network")` `getTestState("network-with-multiple-region-controllers"`.